package.json

```diff
@@ -103,8 +106,14 @@
 		"test": "react-scripts test --env=jsdom",
 		"eject": "react-scripts eject",
 		"lint": "eslint src --ext .js,.jsx",
-		"lint:fix": "eslint src --fix --ext .js,.jsx"
+		"lint:fix": "eslint src --fix --ext .js,.jsx",
+		"commit": "cz"
 	},
+	"config": {
+		"commitizen": {
+		  "path": "./node_modules/cz-conventional-changelog"
```

Comment:
Did you try to simplify path to `"cz-conventional-changelog"`?

Reply:
No, I didn't. I simplify path and it work